### PR TITLE
Update indicator-manage.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that Importing Indicators with specific categories requires the string to be written on Pascal Case convention

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10146

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/indicator-manage.md [hunk 1]
```diff
@@ -56,7 +56,10 @@ Download the sample CSV to know the supported column attributes.
 5. Select **Done**.
 
 > [!NOTE]
-> Only 500 indicators can be uploaded for each batch.
+> Only 500 indicators can be uploaded for each batch. 
+
+> [!NOTE]
+> Attempting to import Indicators with specific categories requires the string to be written on Pascal Case convention and only accepts the category list available at the portal.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/indicator-manage.md]:
```suggestion
>
> Attempting to import indicators with specific categories requires the string to be written in Pascal case convention and only accepts the category list available at the portal.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**